ocs_ci/ocs/ocs_upgrade.py

```diff
@@ -505,6 +505,20 @@ def run_ocs_upgrade(operation=None, *operation_args, **operation_kwargs):
         channel = upgrade_ocs.set_upgrade_channel()
         upgrade_ocs.set_upgrade_images()
         upgrade_ocs.update_subscription(channel)
+        if (config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM) and not (
+            upgrade_in_current_source
+        ):
+            log.info("Sleep 30 mins before linking secret/SAs")
+            time.sleep(30)
```

Comment:
did you meant 30 sec. or the sleep need to be 1800 ?

Reply:
30 secs :) thanks for catching that, will update